### PR TITLE
Docs look and feel update

### PR DESCRIPTION
**Pull request by errordeveloper**
It will make docs appear [just like the website](http://docs.weave.works/weave/docs_lookandfeel/head/index.html).

### CODE REVIEW

File: site/_includes/header-navbaronly.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+              <a href="http://docs.weave.works">Docs</a>
+            </li>
+            <li>
+              <a href="http://github.com/weaveworks">Code</a>
+            </li>
+            <li>
+              <a href="http://blog.weave.works">Blog</a>
+            </li>
+            <li>
+              <a href="http://weave.works/help.html">Help &amp; Support</a>
```
Comment (paulbellamy) [inline site/_includes/header-navbaronly.html]: Should be `http://weave.works/help/index.html`

File: site/_includes/footer.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <div class="row clearfix">
+      <div class="col-md-3 col-md-offset-2 column">
+        <b>Company</b><br>
+        <a href="http://weave.works/about.html">About Weave</a><br>
+        <!-- <a href="#">Meet the Team</a><br> -->
+        <!-- <a href="#">Careers</a><br> -->
+        <a href="http://blog.weave.works">Blog</a><br>
+        <br>
+        <b>Community</b><br>
+        <a href="http://weave.works/help.html#partners">Partners</a><br>
```
Comment (paulbellamy) [inline site/_includes/footer.html]: Link broken

File: site/_includes/footer.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        <a href="http://github.com/weaveworks">Code</a><br>
+      </div>
+      <div class="col-md-3 column">
+        <b>Documentation</b><br>
+        <a href="http://docs.weave.works">Docs</a><br>
+        <a href="http://weave.works/guides/">Getting Started Guides</a><br>
+        <a href="http://weave.works/help.html">Help and Support</a><br>
```
Comment (paulbellamy) [inline site/_includes/footer.html]: Link broken

**Comment by errordeveloper** [comment]
@paulbellamy I have fixed the links to help page.

**Comment by rade** [comment]
shouldn't this be merged into latest_doc_updates?

**Comment by errordeveloper** [comment]
@rade thanks for the reminder!

**Pull request merged**